packages/fast-usdc/src/fast-usdc.contract.js

```diff
@@ -258,7 +261,10 @@ export const contract = async (zcf, privateArgs, zone, tools) => {
     );
   }
 
-  const nobleAccountV = zone.makeOnce('NobleAccount', () => makeNobleAccount());
+  // v1 has `NobleAccount` which we don't expect to ever settle. how can we delete it?
+  const nobleAccountV = zone.makeOnce('NobleAccount2', () =>
```

Comment:
maybe we can give this a name that doesn't reek of "oops"

perhaps with the channel id?
```suggestion
  const nobleAccountV = zone.makeOnce('NobleAccount-62', () =>
```

Reply:
or, `NobleICA`